Framework/CurveFitting/src/FuncMinimizers/FABADAMinimizer.cpp

```diff
@@ -39,6 +39,8 @@ const size_t LOWER_CONVERGENCE_LIMIT = 350;
 const size_t JUMP_CHECKING_RATE = 200;
 // low jump limit
 const double LOW_JUMP_LIMIT = 1e-25;
+// random number generator
+std::mt19937 rng;
```

Comment:
Do we need a way to seed this?

Reply:
I don't think the user would want to set the seed. I can seed it with the current time for more randomness.

Reply:
I think it would be better if we had something that was reproducible on running it again with the set of inputs. I guess we can always add a seed option later if necessary.